source/_integrations/iammeter.markdown

```diff
@@ -13,12 +13,14 @@ ha_platforms:
   - sensor
 ---
 
-`iammeter` provides real-time readings of single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
+`IAMMETER` provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
+[Monitor your solar PV system with IAMMETER energy monitor in Home Assistant](https://community.home-assistant.io/t/monitor-your-solar-pv-system-in-home-assistant-energy-dashboard-vs-energy-management/342710)
```

Comment:
I don't think we should link to community forum resources from our documentation.

Reply:
Hi Frenck:
So glad to receive your message again.
I have removed this content temporarily.
But in fact, we want to introduce some specific solutions that use our products in HA here.
Such as how to monitor the solar PV system? how to use the Grafana to display data while using HA to get the data?
Has it permissible to publish such content in the doc?

If so , could we use the content post in our website or the HA official forum?

Thanks for your attention.
BR
Jason

Reply:
> Such as how to monitor the solar PV system? how to use the Grafana to display data while using HA to get the data?
> Has it permissible to publish such content in the doc?

We are not the Grafana docs. Things like these can be made available on the community forums or your own resources but are not what we want to have in our end-user documentation.

Reply:
OK